pkg/lib/timeformat.js

```diff
@@ -16,9 +23,24 @@ export const time = t => formatter({ timeStyle: "short" }).format(t);
 export const timeSeconds = t => formatter({ timeStyle: "medium" }).format(t);
 // June 30, 2021
 export const date = t => formatter({ dateStyle: "long" }).format(t);
+// 2021-06-30
+export const dateShort = t => formatter().format(t);
```

Comment:
The comment is misleading:
```
> new Intl.DateTimeFormat('en').format()
'7/11/2021'
```
This makes sense, as that *is* the US short date format. YYYY-MM-DD is ISO format. We may use that somewhere some day, but right now we are standardizing around localized date formats.

I.e. the comment should be "06/30/2021"